data-processing/entities/sentences_pdf/commands/fetch_spp_jsons.py

```diff
@@ -36,6 +38,10 @@ class SppJsonTask:
 
 
 class FetchSppJsons(ArxivBatchCommand[ArxivId, None]):
+    def __init__(self, args: Any) -> None:
+        super().__init__(args)
+        self._spp_host_url = os.getenv("S2_SPP_HOST_URL", None)
```

Comment:
We'll want the port as well. The env vars are named:

`SPP_HOST` and `SPP_PORT`